cmd/kops/get_cluster.go

```diff
@@ -121,6 +136,8 @@ func RunGetClusters(context Factory, out io.Writer, options *GetClusterOptions)
 		if err != nil {
 			return err
 		}
+
+		fmt.Fprint(out, get_cluster_full_warning)
```

Comment:
We also suggested adding an annotation, and blocking use of the yaml until the annotation was removed.  Thoughts on this approach vs the annotation approach?